apm-agent-plugins/apm-jdbc-plugin/src/test/java/co/elastic/apm/agent/jdbc/AbstractJdbcInstrumentationTest.java

```diff
@@ -424,9 +428,15 @@ private void assertSpanRecorded(String rawSql, boolean preparedStatement, long e
         assertThat(service.getName().toString()).isEqualTo(expectedDbVendor);
         assertThat(service.getResource().toString()).isEqualTo(expectedDbVendor);
         assertThat(service.getType()).isEqualTo(DB_SPAN_TYPE);
+
+        assertThat(span.getOutcome())
+            .describedAs("span outcome should be explicitly set to either failure or success")
+            .isNotEqualTo(Outcome.UNKNOWN);
```

Comment:
Ideally, we can do that generically in the MockReporter

Reply:
Yes, that will be an opt-in check, as enabling this check by default would lead to too many changes (all plugins that create spans & transactions would be impacted).